byron/ledger/executable-spec/test/Generator.hs

```diff
@@ -0,0 +1,109 @@
+module Generator where
```

Comment:
We should move the functions here to modules that have a more specific name.

Reply:
I asked in your PR #490, but I'll do it here as it is related: how about extracting those generators that you have from a class instance and making them available at the top level? I guess as we go we'll figure out how to organize them into modules, but your suggestion to follow `src` module names makes sense to me.

Reply:
> how about extracting those generators that you have from a class instance and making them available at the top level? 

Sure, if they are generators that other modules could use we should expose them.

> I guess as we go we'll figure out how to organize them into modules,

I'd rather try to distribute generators to modules now, otherwise we might run the risk of people re-writing these generators because they didn't find them.

Reply:
Done! Ledger update generators are now in `src` in the `Ledger.Update.Generator` module.